znoyder/templates/project-config.j2

```diff
@@ -0,0 +1,7 @@
+- project:
```

Comment:
We should use at the beginning of the template:
---

This however breaks the 'a' write mode where '---' is added every time above name.

Reply:
We don't use "---" in our templates, so what am I missing here?

Reply:
We do. I had to add this, see https://github.com/rhos-infra/znoyder/blob/main/znoyder/templates/empty-zuul-project.j2